### ISSUE TITLE
Firefox input sliders are not styled correctly (FIX INCLUDED)

### ISSUE THREAD

**Issue by yankiara** [soflyy/oxygen-bugs-and-features#1130]
**Describe the bug**
Firefox input sliders are not styled correctly.

FIX:
```
		.oxygen-measure-box-slider {
		  top: 2px;
		}
		.oxygen-measure-box-slider input[type="range"] {
		  background: transparent;
		}
		.oxygen-measure-box-slider input[type="range"]::-moz-range-thumb {
		  border-width: 2px;;
		}
```

**Have you tried all the steps at https://oxygenbuilder.com/documentation/troubleshooting/troubleshooting-guide/?**
Yes

**Are you able to replicate the issue on a Sandbox install at https://oxygenbuilder.com/try?**
Dont' need.

**To Reproduce**
1. Open Oxygen with Firefox and click on any element, open left panel

**Expected behavior**
Input sliders like font-size should be OK, like Chrome or other browser.

**What actually happens**
Input sliders are not styled correctly (or have no style).

**Screenshots**
![95291699_819604478567229_5640961625105104896_n](https://user-images.githubusercontent.com/36680757/81469169-d7a8f780-91e3-11ea-954e-84a8975647d7.jpg)

**Desktop (please complete the following information):**
- OS: Windows
 - Browser Firefox
 - Oxygen Version 3.3

**Comment by ddon** [comment]
Just checked most recent Beta 1 version, and this still the case! :)

**Comment by KittenCodes** [comment]
Fixed in v3.8 Beta 1.
